hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/txns/contract/ContractCallTransitionLogicTest.java

```diff
@@ -222,6 +241,26 @@ void verifyExternaliseContractResultCallEth() {
         inOrder.verify(worldState).getCreatedContractIds();
     }
 
+    @Test
```

Comment:
This is a simple test for fails on missing receiver sig req.  But there's no corresponding simple test for _only the condition_ that the same thing _succeeds_ if the receiver sig req is there.  All the other test enable that condition - but it's kind of just there to make the other tests, testing _other_ things, succeed.  

I think it would be nice to have a matched test to this one where the _only_ change is the presence/absence of the receiver sig req flag.  More granular testing of this, IOW.